cli/installer/windows/azd.wixproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why do these files look new in this PR when I've already merged them?

Reply:
Needed to do a rebase